INSTALL.asciidoc

```diff
@@ -45,15 +60,27 @@ cd /tmp/os-autoinst-run
 --------------------------
 7.  create a minimal "vars.json" config file
 +
+.vars.json
+[source,json]
 ---------------------------------------------------------
-cat <<EOF >vars.json
 {
-    "DISTRI":  "sle",
-    "CASEDIR": "/space/SVN/os-autoinst-distri-opensuse/",
-    "BACKEND": "s390x"
+   "ARCH" : "x86_64",   
+   "BACKEND" : "qemu",
+   "CASEDIR" : "/path/to/os-autoinst-distri-opensuse",
+   "CDMODEL" : "virtio-scsi-pci",
```

Comment:
sure, would make sense. I only see the problem that os-autoinst-distri-example 
is less tested and used. But feel free to provide a suggestion how to update 
the documentation.